### ISSUE TITLE
Monochrome launcher icon for Android 13+

### ISSUE THREAD

**Issue by Tommy-Geenexus** [bromite/bromite#2274]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md)
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have searched [existing issues](https://github.com/bromite/bromite/issues) for my feature request. This is a new issue (NOT a duplicate) and is not related to another issue.
- [X] This is a feature request for the Bromite browser; not the website nor F-Droid nor anything else.

### Is your feature request related to privacy?

No

### Is there a patch available for this feature somewhere?

No

### Describe the solution you would like

Android 13 added support for themed launcher icons (https://developer.android.com/about/versions/13/features#themed-app-icons), i'd appreciate if bromite would ship with a monochrome launcher icon for Android 13+.

### Describe alternatives you have considered

None

**Comment by kokukyocho** [comment]
Hello @Tommy-Geenexus!

I am a bot :robot:

Thanks for submitting this issue!
The issue you submitted does not follow the template for either a [bug](https://github.com/bromite/bromite/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml) or a [feature request](https://github.com/bromite/bromite/blob/master/.github/ISSUE_TEMPLATE/feature_request.yml).
Restore the questions that you removed or changed and the issue will be automatically re-opened afterwards.
Please **DO NOT** spam the issue tracker with new issues.

If this is not an issue nor a feature request and you want to start a discussion then you can use [GitHub Discussions](https://github.com/bromite/bromite/discussions).

**Comment by rodrigoswz** [comment]
[Rkicons](https://github.com/RadekBledowski/rkicons) has an amazing Bromite themed icon:

![Bromite by Rkicons](https://user-images.githubusercontent.com/14058427/191073709-ac2581b6-33b7-4c1d-8690-54ff42baa52e.png)

**Comment by Sanjay451** [comment]
Would like to say that, I have been waiting for a long time for Bromite to support Themed app icon. Bromite is the only application in my app drawer which has been there for a long time and also the only app in my app drawer which does not have Themed app icon. 

Never late to say I think, this is the best browser on Android that does block many ads. And had a great support in development. Thanks for all the Devs and to @csagan5 for making this application to run so smooth

Request to provide Themed app icon support to Bromite. 

Thanks.

**Comment by laureanorp** [comment]
I was gonna create an issue for this, including an icon proposal, but the one present on Rkocons a few comments above is perfect. I upvote this request!

**Comment by JaviToro** [comment]
I agree with all of you. Implementing themed icons is not a hard task and it provides a lot of richness to the app.

**Comment by gsture** [comment]
I would also like to request an themed icon. I switched to ungoogled chromium for the time being because bromite was the only icon that was still sticking out and it was giving me an eyesore.

**Comment by mrandr01d** [comment]
I'd also like to add my name to this request. Bromite is the last app on my main homescreen that doesn't have a monochrome icon; I agree that it is quite the eyesore. I tried going back to vanilla chrome just for the icon, but the ads were out of control. Might try Firefox with ublock for a while, but I prefer chromium.
Please implement this ASAP! :)

**Comment by Indianb0y016** [comment]
> I'd also like to add my name to this request. Bromite is the last app on my main homescreen that doesn't have a monochrome icon; I agree that it is quite the eyesore. I tried going back to vanilla chrome just for the icon, but the ads were out of control. Might try Firefox with ublock for a while, but I prefer chromium. Please implement this ASAP! :)

FIrefox unfortunately does not have a monochrome icon yet.

Am also adding my name to this request!

**Comment by coxackie** [comment]
Also pitching in my support on this. It will make the A13 experience so much better.
